lib/matplotlib/axes/_base.py

```diff
@@ -2995,15 +2995,21 @@ def get_xticklabels(self, minor=False, which=None):
     @docstring.dedent_interpd
     def set_xticklabels(self, labels, fontdict=None, minor=False, **kwargs):
         """
-        Set the xtick labels with list of strings *labels*
+        Set the xtick labels with list of strings labels
 
-        Return a list of axis text instances.
+        Parameters
+        ----------
+        labels: list of str
+            list of string labels
 
-        *kwargs* set the :class:`~matplotlib.text.Text` properties.
-        Valid properties are
-        %(Text)s
+        Returns
+        -------
+        list of str
+            A list of axis text instances
 
-        ACCEPTS: sequence of strings
+        Other Parameters
+        -----------------
+        kwargs : :class:`~matplotlib.text.Text` properties.
```

Comment:
It's not about formatting as bold, but the double-asterisks to denote keyword arguments.